helm-ag.el

```diff
@@ -239,15 +239,35 @@
        (read-directory-name "Search directory: " nil nil t))
     default-directory))
 
+(defsubst helm-ag--helm-header (dir)
+  (concat "Search at " dir))
+
+(defvar helm-ag-keymap
+  (let ((map (make-sparse-keymap)))
+    (set-keymap-parent map helm-map)
+    (define-key map (kbd "C-l") 'helm-ag--up-one-level)
+    map)
+  "Keymap for `helm-ag'.")
+
+(defun helm-ag--up-one-level (arg)
+  (interactive "p")
```

Comment:
Where is `arg` used? Is it required by Helm? (I'm yet unfamiliar with how Helm is implemented.)